db/test/test_dataset.py

```diff
@@ -119,6 +123,22 @@ def test_get_by_user_id(self):
         datasets = dataset.get_by_user_id(self.test_user_id, public_only=False)
         self.assertEqual(len(datasets), 2)
 
+    def test_add_recordings(self):
+
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        ds = dataset.get(id)
+
+        dataset.add_recordings(self.test_data_recordings)
+
+    def test_delete_recordings(self):
+
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        ds = dataset.get(id)
+
+        for cls in ds["classes"]:
+            dataset.delete_recordings(cls)
+
```

Comment:
Similar to the add method, you should test that the delete call actually did something